### ISSUE TITLE
Update to 11.0.3 required without it being available

### ISSUE THREAD

**Issue by Forage** [bitpay/copay#11138]
Today I received a notification stating:

-------------------------------------------------------------
**Immediate Action Required!**
To continue making payments with BitPay, please update your app to 11.0.3.
-------------------------------------------------------------

Which is funny because Copay has not seen a single Android update since 8.0.5 in December 2019, unlike BitPay since 9.0.5 in February 2020, despite the commitment to release new versions when major changes have been made (https://github.com/bitpay/copay/issues/9665#issuecomment-610369755).

Does this mean the end of Copay for Android or will a new version be made available in the very near future?

**Comment by lucciisgithub** [comment]
I am terrified that they did this. Now that i updated to 11.0.3, i cant access my funds or import a wallet!! WTF!

**Comment by digil13** [comment]
Need an apk to update, but now afraid to do so based on feedback of user above.

**Comment by lucciisgithub** [comment]
Yes do not update!! Run to another wallet if you still can. I wish i did before i knew this was going to happen.

**Comment by sonrise121** [comment]
I can't find copay App of 11.0.3 I only find 5.3.1..please help I can't afford to loose my money that I have the wallet

**Comment by digil13** [comment]
Network errors seem to be resolved now. Still moving my coins to another wallet.

**Comment by mabelley** [comment]
Just clear your cache in advanced settings and funds will return

**Comment by Daniel-Mendes** [comment]
I'm currently trying to build the apk for you

**Comment by sonrise121** [comment]
Please help no link to download the updated copay... Copay.io. only take me to Googleplay to download bitpay and not copay.. Please help me can't afford to loose my bitcon

**Comment by Forage** [comment]
Calm down guys, you should all have your mnemonic seed words noted somewhere so there's no risk of loosing anything.
It's just a matter of not being able to use the app any more for some tasks, if we are to believe the notification to begin with.

**Comment by Forage** [comment]
Yet another version has just been released without a bitpay or copay Android build. This while still no explanation for the created panic has been given.

How can you keep your users in this uncertainty?
